### PR TITLE
feat: add bc-explorer charts

### PR DESCRIPTION
**Pull request by 0xff-dev**
Fix: https://github.com/kubebb/components/issues/2

### CODE REVIEW

File: charts/bc-explorer/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+type: application
+version: v0.0.1
+icon: https://avatars.githubusercontent.com/u/116370569
+keywords:
+  - explorer
+  - bestchains
+sources:
+  - https://github.com/bestchains/bc-explorer
+maintainers:
+  - name: nkwangleiGIT
```
Comment (bjwswang) [inline charts/bc-explorer/Chart.yaml:13]: only keep 
- 0xff-dev
- bjwswang
- Abirdcfly

File: charts/bc-explorer/values.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+deployImage: hyperledgerk8s/bc-explorer:v20230504
```
Comment (bjwswang) [inline charts/bc-explorer/values.yaml:1]: split into two parts:

- deployment
- runtimeArgs

File: charts/bc-explorer/templates/ingress.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
+apiVersion: networking.k8s.io/v1
```
Comment (bjwswang) [inline charts/bc-explorer/templates/ingress.yaml:1]: check if ingress is enabled

**Review by bjwswang — COMMENTED** [review]
Need some changes

File: charts/bc-explorer/templates/deploy.yaml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+        image:  {{ .Values.deployment.image }}
+        command:
+        - listener
+        args:
+        - -v=5
+        - -injector={{ .Values.runtimeArgs.explorerDB }}
+        - -dsn={{ template "pg-svc" . }}
+        ports:
+        - containerPort: 9999
+      - name: observer
```
Comment (bjwswang) [inline charts/bc-explorer/templates/deploy.yaml:38]: How about this `observer`? Will observer panic ff current cluster do not have ibp resources?

File: charts/bc-explorer/templates/rbac.yaml [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+  - apiGroups:
+      - apiextensions.k8s.io
+    resources:
+      - customresourcedefinitions
+    verbs:
+      - get
+  - apiGroups:
+      - ""
+    resources:
+      - pods
```
Comment (bjwswang) [inline charts/bc-explorer/templates/rbac.yaml:21]: remove unused rules

File: charts/bc-explorer/templates/deploy.yaml (hunk 4 above)
Comment (Abirdcfly) [reply charts/bc-explorer/templates/deploy.yaml:38]: If the current cluster do not have ibp resources, the observer will log error, but no panic
```bash
E0609 14:01:45.412709   26897 reflector.go:138] github.com/IBM-Blockchain/fabric-operator/pkg/generated/informers/externalversions/factory.go:119: Failed to watch *v1beta1.Channel: failed to list *v1beta1.Channel: the server could not find the requested resource (get channels.ibp.com)
...
```

**Review by Abirdcfly — APPROVED** [review]

File: charts/bc-explorer/templates/deploy.yaml (hunk 4 above)
Comment (bjwswang) [reply charts/bc-explorer/templates/deploy.yaml:38]: 🆗 then it will be fine

**Review by bjwswang — APPROVED** [review]

**Pull request merged**